src/sbuf.c

```diff
@@ -1235,6 +1236,44 @@ static bool setup_tls(struct tls_config *conf, const char *pfx, int sslmode,
 	return true;
 }
 
+static bool tls_config_changed(struct tls_config *new_server_connect_conf)
```

Comment:
Is the struct sufficiently stable or only have binary comparable items in it such that we could simply mem compare? Or do we need to list out all of the attributes?

Reply:
Interesting thought... I suspect that it's stable enough for that kind of comparison. I'll try it.

Reply:
aha yeah @jcoleman and I chatted and this doesn't work with a couple pointers to other structs and strings in `tls_config`